packages/elastic_agent/changelog.yml

```
@@ -1,4 +1,9 @@
 # newer versions go on top
+- version: "1.4.2"
```

```suggestion
- version: "1.5.0"
```

I think it is probably worth bumping to 1.5.0 since we are adding new functionality here.

---

Thanks! Updated to `1.5.0`